app/src/main/java/com/example/plantsapp/presentation/ui/plantcreation/PlantCreationViewModel.kt

```diff
@@ -31,12 +35,16 @@ class PlantCreationViewModel(
                 Plant(
                     Plant.Name(plantName),
                     speciesName,
-                    "",
+                    selectedPicture.value ?: Uri.EMPTY,
```

Comment:
Here is a tricky part: look how your Plant picture is defined, and look what lies behind Uri.EMPTY (answer: null)
So for now I suggest explicitly use null instead of Uri.EMPTY and change imagePlant type tu Uri?